cmd/erasure-server-pool.go

```diff
@@ -215,6 +199,22 @@ func newErasureServerPools(ctx context.Context, endpointServerPools EndpointServ
 		break
 	}
 
+	// initialize the object layer.
+	setObjectLayer(z)
```

Comment:
Please don't set the object layer right here.

Reply:
Healing in general should be executed after the object layer is initialized, in this case many if not all heal MRF entries will be unsuccessfully retried to be healed

I can move this code outside this function anyway but I am not sure, this change does not seem to break anything

Reply:
Okay